lynx/common/actions/chop.py

```diff
@@ -23,19 +23,18 @@ def apply(self, scene: 'Scene') -> None:
         objects_on_square = scene.get_objects_by_position(target_position)
         # TODO put 'Tree' into an Enum
         chopped_object: Object = \
-        list(filter(lambda object_on_square: object_on_square.name == 'Tree', objects_on_square))[0]
+            list(filter(lambda object_on_square: object_on_square.name == 'Tree', objects_on_square))[0]

```

Comment:
Make in-line with `chopped_object: Object =` i.e.
`chopped_object: Object = list(filter(lambda object_on_square: object_on_square.name == 'Tree', objects_on_square))[0]`
Same for other lines that fit in 150 signs.